src/app/services/game-play.service.ts

```diff
@@ -54,6 +54,24 @@ export class GamePlayService {
     return this.game$.getValue();
   }
 
+  public getBoard$(rows: number): Observable<string[]> {
+    return combineLatest([this.acceptedGuesses$, this.currentGuess$, this.currentGuessIdx$]).pipe(
+      map(([acceptedGuesses, currentGuess, currentGuessIdx]) => {
+        const guesses = [];
+        for (let i = 0; i < rows; i++) {
+          if (i === currentGuessIdx) {
+            guesses.push(currentGuess)
+          } else if (!acceptedGuesses[i]) {
+            guesses.push("")
+          } else {
+            guesses.push(acceptedGuesses[i])
+          }
+        }
+        return guesses;
+      })
+    );
+  }
+
```

Comment:
I'd be curious to know what people think of the implementation here? I was worried about `GamePlayService` and `BoardComponent` knowing too much about each others implementation details, so I welcome any feedback about opportunities to inject dependencies in either class.